tests/application_tests/test_env_var_changes.py

```diff
@@ -124,16 +182,41 @@ def test_multiple_env_vars(self):
         # Assert correct replicas
         # Make sure new index has 4 replicas
         assert self.client.get_index(self.index_name_1).get_settings() \
-            ["number_of_replicas"] == 4
-        
+                   ["number_of_replicas"] == 4
+
         # Assert correct EF const
         assert self.client.get_index(self.index_name_1).get_settings() \
-            ["index_defaults"]["ann_parameters"]["parameters"]["ef_construction"] == 5000
+                   ["index_defaults"]["ann_parameters"]["parameters"]["ef_construction"] == 5000
 
         # Assert correct models
         res = self.client.get_loaded_models()
         assert set([item["model_name"] for item in res["models"]]) == set(new_models)
 
+        # ## Testing log output when LEVEL=debug ##
+        #    we want to ensure that  in debug mode, no information is hidden
+
+        # use the index to generate more log outputs (specifically regarding HTTPS requests)
+        self.client.index(self.index_name_1).add_documents(
+            documents=[{'Title': 'Recipes for hippos'}],
+            tensor_fields=['Title'],
+            auto_refresh=True
+        )
+        self.client.index(self.index_name_1).search('something')
+        log_blob = utilities.retrieve_docker_logs(container_name='marqo')
+        print('LOG BLOB')
+        print(log_blob)
+        print('END LOG BLOB')
+        assert 'torch==' in log_blob
+        # assertions about HTTP requests aren't reliable for some reason
+        # assert ((
+        #                 "Unverified HTTPS request is being made to host 'host.docker.internal'. "
+        #                 "Adding certificate verification is strongly advised." in log_blob)
+        #         or (
+        #                 "Unverified HTTPS request is being made to host 'localhost'. "
+        #                 "Adding certificate verification is strongly advised." in log_blob))
+        assert 'Status: Downloaded newer image for marqoai/marqo-os' in log_blob

```

Comment:
ditto my comment about the other startup steps

Reply:
done